$OpenBSD: patch-kmymoney_dialogs_settings_CMakeLists_txt,v 1.1 2017/08/24 09:31:35 dcoppa Exp $

commit b0a68ca075928aa29ee1e0007bfb14d714f5a948
Author: Ralf Habacker <ralf.habacker@freenet.de>
Date:   Thu Aug 10 13:45:04 2017 +0200

Fix 'Compile error on Fedora 26'

CMake build system used some outdated variables which has been replaced
by cmake built-in variables to fix the issue.

* Unbreaks Ninja builds too

Index: kmymoney/dialogs/settings/CMakeLists.txt
--- kmymoney/dialogs/settings/CMakeLists.txt.orig
+++ kmymoney/dialogs/settings/CMakeLists.txt
@@ -1,12 +1,12 @@
 add_custom_command(
-  OUTPUT ${KMyMoney2_BINARY_DIR}/kmymoneysettings.h  ${KMyMoney2_BINARY_DIR}/kmymoneysettings.cpp
-  DEPENDS ${KMyMoney2_SOURCE_DIR}/kmymoney.kcfg ${KMyMoney2_SOURCE_DIR}/kmymoneysettings.kcfgc
-  COMMAND ${KDE4_KCFGC_EXECUTABLE} -d ${KMyMoney2_BINARY_DIR}/ ${KMyMoney2_SOURCE_DIR}/kmymoney.kcfg ${KMyMoney2_SOURCE_DIR}/kmymoneysettings.kcfgc
+  OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/kmymoneysettings.h  ${CMAKE_CURRENT_BINARY_DIR}/kmymoneysettings.cpp
+  DEPENDS ${CMAKE_SOURCE_DIR}/kmymoney/kmymoney.kcfg ${CMAKE_SOURCE_DIR}/kmymoney/kmymoneysettings.kcfgc
+  COMMAND ${KDE4_KCFGC_EXECUTABLE} -d ${CMAKE_CURRENT_BINARY_DIR}/ ${CMAKE_SOURCE_DIR}/kmymoney/kmymoney.kcfg ${CMAKE_SOURCE_DIR}/kmymoney/kmymoneysettings.kcfgc
   )
 
 
 set (libsettings_a_SOURCES
-  ${KMyMoney2_BINARY_DIR}/kmymoneysettings.h
+  ${CMAKE_CURRENT_BINARY_DIR}/kmymoneysettings.cpp
   ksettingscolors.cpp
   ksettingsfonts.cpp
   ksettingsforecast.cpp
